src/machine/board_pico.go

```diff
@@ -37,3 +37,20 @@ const (
 	// Onboard crystal oscillator frequency, in MHz.
 	xoscFreq = 12 // MHz
 )
+
+// SPI default pins
+const (
+	// Serial Clock Bus 0 for SPI communications
+	Default_SPI0_SCK = GPIO18
```

Comment:
It would be better to use the same naming convention for these e.g. `SPI0_SCK_PIN` just for consistency's sake. If we want to do something like change the naming, best to do that to all boards at the same time in a PR specific to that idea so we can get agreement from the community.

Reply:
@deadprogram Consider it done!